correlation_toolbox/helper.py

```diff
@@ -64,19 +63,15 @@ def sort_gdf_by_id(data, idmin=None, idmax=None):
     assert((idmin is None and idmax is None)
            or (idmin is not None and idmax is not None))
 
-    if len(data) > 0:
-        # get neuron ids
-        if idmin is None and idmax is None:
-            ids = np.unique(data[:, 0])
-        else:
-            ids = np.arange(idmin, idmax+1)
-        srt = []
-        for i in ids:
-            srt.append(np.sort(data[np.where(data[:, 0] == i)[0], 1]))
-        return ids, srt
+    # get neuron ids
+    if idmin is None and idmax is None:
+        ids = np.unique(data[:, 0])
     else:
-        print('CT warning(sort_spiketrains_by_id): empty gdf data!')
```

Comment:
agreed, it should be possible to call it with empty gdfs, but maybe still emit a warning?